tests/fsharp/FSharpSuite.Tests.fsproj

```diff
@@ -72,6 +72,7 @@
     <Compile Include="Compiler\Language\ComputationExpressionTests.fs" />
     <Compile Include="Compiler\Language\StructActivePatternTests.fs" />
     <Compile Include="Compiler\Language\TypeDirectedConversionTests.fs" />
+    <Compile Include="compiler\language\ComputedCollectionLoweringTests.fs" />
```

Comment:
Mind the casing here, such things can cause weird git issues.

Reply:
Hmm, good catch. I've seen VS do this a couple times.

Reply:
09f4023